_sass/_post.scss

```diff
@@ -30,3 +30,7 @@
   }
 
 }
+
+p img {
```

Comment:
Awesome, @thunsaker can you move this to [.post-content](https://github.com/gdg-managua/jekyll-mdl/blob/master/_layouts/post.html#L13) class